resctrl/utils.go

```diff
@@ -105,7 +108,7 @@ func prepareMonitoringGroup(containerName string, getContainerPids func() ([]str
 		return rootResctrl, nil
 	}
 
-	pids, err := getContainerPids()
+	pids, err := getPids(containerName)
```

Comment:
Is `getContainerPids()` used (it does not seem to be)? If not, would it be possible to drop the argument?

Reply:
It should be, but it touches quite a bunch of definitions. I'll follow up on this if that works for you:)